src/model/user.model.ts

```diff
@@ -43,6 +37,34 @@ const UserSchema = new Schema<IUser, UserModel, IUserMethods>(
         message: (props) => `${props.value} is not a valid password`,
       },
     },
+    realName: {
+      type: Schema.Types.String,
+    },
+    birthday: {
+      type: Schema.Types.String,
+    },
+    gender: {
+      type: Schema.Types.String,
+      enum: Gender,
+    },
+    age: {
+      type: Schema.Types.Number,
+    },
+    introduction: {
+      type: Schema.Types.String,
+    },
+    phoneAreaCode: {
```

Comment:
phoneAreaCode and phoneNumber should be string,otherwise the first '0' will disappear

Reply:
nice, add a validator for checking number as well